src/main/java/cow/tasks/Deadlines.java

```diff
@@ -0,0 +1,75 @@
+package cow.tasks;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import cow.exceptions.MissingParametersException;
+
+/**
+ * Deadlines Task class.
+ */
+public class Deadlines extends Task {
```

Comment:
It's better if the class name is single-valued since it's a blueprint for a deadline task

```suggestion
public class Deadline extends Task {
```